app/main.py

```diff
@@ -1,34 +1,103 @@
-class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+# class Deck:
+#     def __init__(self, row, column, is_alive=True):
+#         pass
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
+    decks = []
+
+    def __init__(self,
+                 start: tuple,
+                 end: tuple,
+                 is_drowned: bool = False) -> None:
         # Create decks and save them to a list `self.decks`
-        pass
+        self.deck = self.create_deck(start, end)
+        Ship.decks.append(self)
+
+    @staticmethod
+    def create_deck(start: tuple, end: tuple) -> list:
+        if start[0] == end[0]:
+            return [(start[0], point) for point in range(start[1], end[1] + 1)]
+
```

Comment:
```suggestion
```
Sry for this, but it seems pretty simple, so why should you add this space, can't get it.